doc/underpinnings.tex

```diff
@@ -0,0 +1,779 @@
+data for each predicted calendar year.
+
+As we will come to see the algorithmic approach to working out the
+$t$'s in $Y_n$ introduces stochastic processes. Thus we introduce the
+need to use Monte Carlo methods to ensure confidence in our
+calculations.
+
+The model can be thought of as a simulation in with a simulation clock
+that ticks once a calendar year. On clock tick all the entities have
+their new population calculated: based on their existing state,
+pre-calculated historic probability distributions and projected
+population.
+
+\subsection{Send Model's Population Prediction Algorithm}
+
+The algorithm can be simplified to:
+
+$Y_{n+1} = T(Y_n)$
+
+$Y_n$ represents calendar year n's population state, $T$ the function
+that calculates the next years population state from it.  The
+simplification of $T$ means that some things that are variable are
+closed over i.e, the external ONS population data is not a variable,
+nor are the pre-calculated probability distributions.
+
+Let us consider first a single entity's population $t_{s_x,n_y,ay_z}$.
+How will $T$ operate on this?
+
+\subsubsection{Movers,  Non-Movers, Leavers and Joiners}
+
+The model introduces the concepts of movers($m$), leavers($l$), and
+joiners($j$).
+
+Let $m_{s_x,n_y,ay_z}$ be defined as all the pupils that move
+\textbf{into} a particular entity from another entity.
+
+Let $n_{s_x,n_y,ay_z}$ be defined as all the pupils that
+\textbf{don't} move out of a particular entity (non-movers or
+stayers).
+
+Let $l_{s_x,n_y,ay_z}$ be defined as all the pupils that
+\textbf{leave} a particular entity and out of the Send programme
+completely.
+
+Let $j_{s_x,n_y,ay_z}$ be defined as all the pupils that join
+\textbf{into} a particular entity from the outside populace.
+
+
+So $T(Y_n)_{s_x,n_y,ay_z} = m_{s_x,n_y,ay_z} + n_{s_x,n_y,ay_z} -
+l_{s_x,n_y,ay_z} + j_{s_x,n_y,ay_z} $
+
+Reading this equation aloud makes obvious intuitive sense.  The
+question now becomes how do we calculate each of the components of
+this function?
+
+In all the following sections except where noted it is assumed that
+the underlying processes governing leavers, joiners, and movers are
+independent and identically distributed meaning that we can combine
+historic calendar years data as one.
+
+\subsubsection{Leavers}
+
+From our historic data we can work out the percentages over the
+calendar years of how many pupils leave a particular entity,
+$(s_1,n_1,ay_1)$.  Averaging this sets our expectation e.g. for
+$(s_1,n_1,ay_1)$ we may expect $40\%$ of the population to leave.
+
+For the purposes of our simulation we do not want every run to have
+exactly $40\%$ of the population leave but rather have some variance
+in the size.  How these variances in the population propagate through
+the simulation lets us see confidence bounds when we come to
+aggregate the results of many runs of the model.
+
+A first pass at introducing variance is to use the binomial
```

Comment:
Clarify what the difference is between what you're doing with the binomial and what you're doing with the beta. 
IIUC the beta is to create a "known" true underlying distribution in each run, then the binomial is to do some actual draws from that distribution?

Reply:
hmmm, the beta distribution is only used to create the probability value for the binomial distribution. We draw from the beta distribution once to get a value for our probability, then the binomial is drawn once with this probabilty and population to get a number of pupils.

Reply:
This section is supposed to explain exactly what you are asking for clarification of.